config/development.json

```diff
@@ -30,6 +30,7 @@
 	"zendesk_support_chat_key": "715f17a8-4a28-4a7f-8447-0ef8f06c70d7",
 	"features": {
 		"ad-tracking": false,
+		"a4a-dev-sites-referral": true,
```

Comment:
I would recommend we keep the flag `true` in development only, at least until after we launch the free A4A dev sites. It may cause unnecessary ambiguity when testing things internally before the launch.

Reply:
Done.